perf/benchmark/configs/istio/security_authz/latency.yaml

```diff
@@ -0,0 +1,18 @@
+# Data: latency
+# Config: security_authz
+telemetry_mode: "none"
+conn:
+    - 2
+qps:
+    - 1000
+duration: 240
+perf_record: true
+# we only care about both sidecar mode, which is the base line for mTLs overhead comparison.
+run_bothsidecar: true
+run_serversidecar: false
+run_clientsidecar: false
+run_baseline: false
+
+extra_labels: "security_authz"
```

Comment:
for now I'm wondering should we include the params of the generated policy in the extra label? e.g. `security_authz_2000_policy_1000_source_ip_1000_path`?

In the future, when the policy generation param is configurable in the yaml file directly the extra label should probably automatically include them.